gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Dec 24 21:00:34 CAT 2015
+#Fri Jan 27 18:22:28 EST 2017
```

Comment:
Do we need this line?

Reply:
It's a copy/paste. I'll remove it. It was probably auto generated by whatever gradle script made what I copied.